mr/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -167,8 +176,17 @@ public static Table table(Configuration config, String name) {
    * @param config The configuration used to get the data from
    * @return The collection of the table names as returned by TableDesc.getTableName()
```

Comment:
Can you please update the javadoc to reflect that now we are returning catalogName-tableName pairs?

Reply:
Good catch! Fixed it.